stack/templates/deployment.yaml

```diff
@@ -92,12 +92,14 @@ spec:
         {{- end }}
       initContainers:
       {{- range $i, $container := .Values.initContainers }}
-      {{- $imageDict := fromYaml (include "initContainer.image" (dict "Chart" $global.Chart "Values" $container)) }}
+      {{- with (dict "Chart" $global.Chart "Values" $container) -}}
+      {{- $imageDict := fromYaml (include "image" .Values) }}
       {{- $container = mergeOverwrite $container $imageDict }}
       {{- with omit $container "envFrom" "env" }}
         - {{- toYaml . | nindent 10 }}
           {{- include "service.configuration" $service | nindent 10}}
       {{- end }}
+      {{- end -}}
```

Comment:
Is the `Chart` from this dict used anymore? You might be able to remove `{{- with (dict "Chart" $global.Chart "Values" $container) -}}` and just pass `$container` to `include "image"`